### PR TITLE
[Bugfix] Fix trigger the FULFILLMENT_APPPROVED webhook for partial fulfillments

### PR DESCRIPTION
**Pull request by d-wysocki**
Now the  `FULFILLMENT_APPPROVED` webhook is triggered only when all fulfillments are approved/fulfilled, this PR fix this and allow to trigger this webhook for partial fulfillments, even if not all fulfillments are fulfilled.

Tested this changes locally and works as expected.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [x] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Review by maarcingebala — APPROVED** [review]

**Review by korycins — APPROVED** [review]

File: saleor/order/actions.py [hunk 1]
```diff
@@ -482,9 +482,9 @@ def approve_fulfillment(
         update_order_status(order)
 
         call_event(manager.order_updated, order)
+        call_event(manager.fulfillment_approved, fulfillment)
```
Comment (fowczarek) [inline saleor/order/actions.py:485]: Why did you remove `transaction.on_commit`?

**Review by fowczarek — APPROVED** [review]
